bitnami/prometheus-operator/templates/prometheus/service.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.prometheus.service.type }}
+  {{- if index .Values.prometheus "stickySessions" }}
+  sessionAffinity: ClientIP
+  {{- end }}
```

Comment:
You need to add this `prometheus.stickySessions` parameter to the **values.yaml** and the **values-production.yaml**